setup.cfg

```diff
@@ -32,4 +32,8 @@ exclude_lines =
 
 [kolibri:i18n]
 project = kolibri
+project_id = 201933
 locale_data_folder = kolibri/locale
+#
+# From the kolibri-tools package
+plugins_path = ../../build_tools/build_plugins.txt
```

Comment:
I don't know what this file path is relative to, but it's not relative to the `setup.cfg`.

For clarity, and consistency with the `locale_data_folder` setting, we should make this `build_tools/build_plugins.txt` and handle that appropriately by resolving the path relative to the `setup.cfg`.

Do we need the locale_data_folder setting here any more, if we are deriving it from the build_plugins.txt?